libs/server/src/lib/utils/read-schematic-collections.ts

```diff
@@ -211,7 +212,12 @@ export async function readSchematicOptions(
       collectionPackageJson.json.generators,
     dirname(collectionPackageJson.path)
   );
-  const schematics =  collectionJson.json.schematics || collectionJson.json.generators;
+  const schematics = Object.assign(
```

Comment:
Same thing here. We should check if the workspace is one or the other, and only read the appropriate schemas.